SparkPost/2web2ui
# EO-847: Updated styling of recipient validation single results

[EO-847](https://jira.int.messagesystems.com/browse/EO-847)
### What Changed
 - Tabs can now be reached directly by route
 - Updated styling on recipient validation page

### How To Test
 - Navigate to `/recipient-valdiation`
     - Should reroute to `/recipient-validation/list`
- Check that tabs work
- Check that single address form still works
- Check that results display

### Notes
 - API is not updated yet so will not show icons and other items.

### To Do
- [x] Move request into single response form on load
- [ ] Tag new routes in Pendo
